templates/Project/project_details.html.twig

```diff
@@ -41,6 +41,29 @@
       } %}
     </div>
   </div>
+  {% if logged_in %}
+  <div class="btn btn-danger d-lg-block col-lg-3 mt-2">
+        {% include 'components/redirect_button.twig' with {
+        'id': 'projectStealButton',
+        'url': path('project_steal', {'id': project.id }),
+        'icon': 'person',
+        'text': "Steal"
+      } %}
+      <aside class="mdc-snackbar">
+        <div class="mdc-snackbar__surface" role="status" aria-relevant="additions">
+          <div class="mdc-snackbar__label" aria-atomic="false">
+            Can't send photo. Retry in 5 seconds.
```

Comment:
Since you have not used the translation system, just a heads up for the future:

`codeview.title` is registered by the developer in the `translations/catroweb.en.yaml` file as 
```
codeview:
    title: Steal
```

All other translation files; E.g `catroweb.de.yml` are translated in its own process (automated by GitHub Actions) using Crowdin -> More details here: https://catrobat.atlassian.net/wiki/spaces/CATWEB/pages/74383377/Crowdin+-+Translation+Management